lib/formats/sh.sh

```diff
@@ -255,19 +255,19 @@ remove_cmd() {
     echo "${COL_YELLOW}Deleting: ${COL_RESET}$1"
     if original_check "$1" "$2"; then
         if [ -z "$DO_DRY_RUN" ]; then
-            if [ ! -z "$DO_KEEP_DIR_TIMESTAMPS" ]; then
-                touch -r "$(dirname $1)" "$STAMPFILE"
+            if [ -n "$DO_KEEP_DIR_TIMESTAMPS" ]; then
+                touch -r "$(dirname "$1")" "$STAMPFILE"
```

Comment:
Do nested quotations like this work? I have some doubts here. :)

Reply:
What is the source of your doubt? The quotes are required inside a subshell because the substitution of `$1` is vulnerable to error from filenames with whitespace, newlines etc.

Reply:
Hmm, I thought this would break due the surrounding quotes, but I had a quick try in the shell and it worked. So consider those doubts as cleared.